styleguide/derek/view-templates/stories/_stories.scss

```diff
@@ -1,5 +1,7 @@
 // Customer Stories Feature Colors
 $default-bg-img: 'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/IMACbanners/collaboration-2.jpg';
+$default-grid-bg: 'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/stories/stories-grid-default.png';
```

Comment:
#494 is gonna change where this file needs to live (and I imagine that's getting merged in today or tomorrow).  It shouldn't be on the CDN but rather in the repo itself.